### DISCUSSION TITLE
Smalltalk syntax support

### DISCUSSION THREAD

**Discussion by sebastianconcept** [massCodeIO/massCode#42] [Ideas]
So far I'm doing this with macOS notes and this looks like something I was looking for! I'm starting to use it and I'd like to include my Smalltalk snippets, how highlight is done? Can we add Smalltalk support for it? Thanks!

**Comment by antonreshetov** [comment]
@sebastianconcept In v3 added Smalltalk support
![image](https://user-images.githubusercontent.com/17480987/183039003-f3fd2ce1-20e2-4a30-8aef-7c0471d46347.png)

**Reply by sebastianconcept** [reply]
@antonreshetov you rock!
